test/testdrive/protobuf-import.td

```diff
@@ -97,14 +109,17 @@ $ schema-registry-publish subject=importee.proto schema-type=protobuf references
 $ schema-registry-publish subject=testdrive-import-csr-${testdrive.seed}-value schema-type=protobuf references=empty.proto,importee.proto
 \${importer-schema}
 
-# TODO(benesch): There is a big bug here! In the Confluent toolchain, these
-# messages are emitted with the CSR magic at the start. We are *not* doing that
-# here, which means that all we are testing is that our incorrectly-formatted
-# Kafka topic is ingestible. We need to teach `kafka-ingest` to accept a
-# `confluent-schema-id` parameter that prefixes these messages with the magic
-# prefix, then teach the ingestor to skip that magic prefix when the CSR is
-# the source of the schema. See #9250.
-$ kafka-ingest topic=import-csr format=protobuf descriptor-file=import.pb message=.Importer
+$ schema-registry-publish subject=testdrive-import-csr-other-${testdrive.seed}-value schema-type=protobuf references=empty.proto,importee.proto
+\${importer-schema}
+
+$ schema-registry-publish subject=testdrive-import-csr-other2-${testdrive.seed}-value schema-type=protobuf references=empty.proto,importee.proto
+\${importer-other-schema}
+
+$ schema-registry-publish subject=testdrive-import-csr-other3-${testdrive.seed}-value schema-type=protobuf references=empty.proto,importee.proto
+\${importer-schema}
+
+# schema id 4 is the correct schema
```

Comment:
Hrm, this doesn't seem stable, unfortunately. The schema ID increases monotonically from the beginning of time. If running against a non-fresh schema registry you'll get a very different schema ID, for example.